docker/Dockerfile

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG CUDA=11.0
+# need 11.1 because tensorflow 2.5.0 requires libcusolver-11-1, not 11-0.
```

Comment:
Do lot leave this comment in the code.

Reply:
Thank you for your suggestion. I'll remove it.